build.sh

```diff
@@ -69,6 +69,19 @@ initDistroRid()
     fi
 }
 
+initTargetDistroRid()
```

Comment:
`$__DistroRid` is used in https://github.com/hseok-oh/coreclr/blob/5145193d4970eaf9a008014a7b02307611bd96ef/build.sh#L237 before nuget packaging, and this function need host environment information to use msbuild on .net core.